test/test_meta.py

```diff
@@ -405,6 +405,7 @@ def run_meta_crossref(
 meta_function_expected_failures = {
     torch.Tensor.item: {b8, bf16, c128, c64, f16, f32, f64, i16, i32, i64, i8, u8},  # aten::_local_scalar_dense
     torch.Tensor.to_sparse: {b8, bf16, f16, f32, f64, i16, i32, i64, i8, u8},  # aten::to_sparse, aten::to_sparse.sparse_dim
+    torch.arange: {bf16, f16, f32, f64, i16, i32, i64, i8, u8},
```

Comment:
Why does this introduce `arange` failures ?

Reply:
Running into the error: `RuntimeError: device type cuda does not match device type of out parameter (meta)` - could it be because arange does not take any tensor inputs?